app/locale/en.js

```diff
@@ -6233,6 +6233,43 @@ module.exports = {
       get_full_course: 'Get Full Course',
       try_sample_lesson: 'Try Sample Lesson',
       trophy_header: 'Level up as a Computer Science Teacher with our Professional Development offerings!'
+    },
+    junior_page: {
+      testimonials_1_quote: 'The future is programmable and you will need to be able to code to participate…I think CoCo [CodeCombat] has the best CS curriculum on the market… the teacher tool kit is what sets it apart.',
```

Comment:
We can fix up the formatting a bit here.
```suggestion
      testimonials_1_quote: 'The future is programmable and you will need to be able to code to participate… I think CodeCombat has the best CS curriculum on the market… the teacher toolkit is what sets it apart.',
```